website/docs/howto.md

```diff
@@ -80,5 +80,6 @@ Note the `match` field, which defines the scope of objects to which a given cons
    * `excludedNamespaces` is a list of namespace names. If defined, a constraint will only apply to resources not in a listed namespace.
```

Comment:
Should we update `excludedNamespaces` and `namespace` as well to include support for prefix-based glob?

Reply:
Added!  Good call.